keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md

```diff
@@ -15,8 +15,9 @@ This sample illustrates how to use [Azure Spring Boot Starter Key Vault Certific
 
 This sample should work together with [azure-spring-boot-sample-keyvault-certificates-server-side].
 
-## Getting started
+This sample works for Linux/Unix OS.
```

Comment:
can this sample works for PC?

Reply:
I think you want to say, the steps in the sample  , using powershell to create resource is not working in PC‘s environment, right?

Reply:
Can't work for Windows power shell.

Maybe `This script can only run by bash shell` is better.